comm/src/test/scala/coop/rchain/p2p/URIParseSpec.scala

```diff
@@ -8,9 +8,11 @@ class URIParseSpec extends FlatSpec with Matchers {
     Left(ParseError(s"bad address: $s"))
 
   "A well formed rnode URI" should "parse into a PeerNode" in {
-    val uri = "rnode://abcde@localhost:12345"
+    val uri = "rnode://abcdef@localhost:12345"
```

Comment:
why `f` was added? is it important for the test?

Reply:
yes, it must be an even number of characters to have bytes (here 3 bytes)

Reply:
👍